content/kubermatic/v2.22/installation/local-kkp-installation/_index.en.md

```diff
@@ -0,0 +1,83 @@
++++
```

Comment:
This is the wrong place for the docs since the feature doesn't exist in v2.22. It should be moved to `content/kubermatic/main` so I can sync it from there into my PR over at #1482.